docs/tables/digitalocean_database.md

```diff
@@ -79,3 +79,13 @@ group by
 order by
   count desc;
 ```
+### Get database firewall trusted sources
+
+```sql
+select 
+  firewall ->> 'type' as "Firewall Source",
```

Comment:
Would it also be good to include the database name as a column? Else I imagine it may be difficult to tell which rules belong to which database